examples/demo/functionplotter.py

```diff
@@ -22,18 +22,25 @@ class PlotExample(HasTraits):
     plot = Instance(Component)
     numpoints = Int(500)
 
+    low_mode = Enum("value", "track")
+    high_mode = Enum("value", "track")
+
+
     traits_view = View(
         Group(
             Item('plot', editor=ComponentEditor(), show_label=False),
             HGroup(
                 HGroup(
-                    Item('object.plot.x_mapper.range.low_setting', label='Low'),
-                    Item('object.plot.x_mapper.range.high_setting', label='High'),
+                    Item('object.plot.x_mapper.range.low_setting', label='Low', editor=TextEditor(), visible_when='object.low_mode == "value" ', ),
```

Comment:
I'm probably being very nitpicky and i apologize for it in advance.

Can you conform to the usual line length limits? We probably don't enforce line length limits in the codebase in general, if not the examples scripts specifically but IMO it's always a good idea to maintain standards (because examples is what other will learn from).

Reply:
You're right. Will conform to line length for this and future changes.